src/main.c

```diff
@@ -562,13 +570,21 @@ main(int argc, char** argv)
 #endif
          errx(1, "%s (file <%s>)", message, frontend_users_path);
       }
-      else if (ret == PGAGROAL_CONFIGURATION_STATUS_CANNOT_DECRYPT
-               || ret == PGAGROAL_CONFIGURATION_STATUS_KO)
+      else if (ret == PGAGROAL_CONFIGURATION_STATUS_KO)
+      {
+         snprintf(message, MISC_LENGTH, "Invalid master key file");
```

Comment:
Why "invalid master key file"? This branch is for configuration errors on frontend user file.